src/config/SectionConfig.ts

```diff
@@ -24,7 +24,7 @@ const sections: SectionConfig = {
     component: B,
   },
   C: {
-    title: "Data Access & Disease Information",
+    title: "Data Access and Disease",
```

Comment:
I would also confirm she meant to use "and" instead of ampersand. There's a mix of different usage throughout the revised tickets.

Reply:
Confirmed it is "and" instead of "&".